src/services/get-songs.ts

```diff
@@ -199,7 +199,7 @@ export default class {
       case 'album': {
         const uri = parsed as spotifyURI.Album;
 
-        const [{body: album}, {body: {items}}] = await Promise.all([this.spotify.getAlbum(uri.id), this.spotify.getAlbumTracks(uri.id, {limit: 50})]);
+        const [{body: album}, {body: {items}}] = await Promise.all([this.spotify.getAlbum(uri.id), this.spotify.getAlbumTracks(uri.id, {limit})]);
```

Comment:
The maximum value the limit parameter can take [is 50](https://developer.spotify.com/documentation/web-api/reference/#/operations/get-an-albums-tracks). I find it pretty unlikely that an album has more tracks than that.

If you want to support albums with more than 50 tracks, you'll have to implement pagination.